provisioning/roles/php/vars/Debian-jessie-php7.x.yml

```diff
@@ -1,3 +1,4 @@
+# debian 8, 2015-04-26
```

Comment:
Please add the mention "released on" next to the date so we know what it is (same for the Debian stretch vars file).